frontend/public/components/utils/name-value-editor.jsx

```diff
@@ -66,8 +66,10 @@ export const NameValueEditor = withDragDropContext(class NameValueEditor extends
     });
     return <React.Fragment>
       <div className="row">
+        <div className="col-md-1 col-xs-1 co-empty__header"></div>
```

Comment:
I think you can drop the co-md-* classes since the col-xs-* classes have the same values (here and below).